Validator/Unique.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Vanio\DomainBundle\Validator;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class Unique extends Constraint
+{
+    const NOT_UNIQUE_ERROR = '';
```

Comment:
Tady nejaky UUID asi, podobne jako to maji oni... Treba nejaky vezmi tady https://www.uuidgenerator.net/version4